components/icon/stories/icon.mdx

```diff
@@ -0,0 +1,57 @@
+import { Canvas, ArgTypes, Meta, Description, Title } from '@storybook/blocks';
+
+import * as IconStories from './icon.stories';
+
+<Meta of={IconStories} title="Docs" />
+
+<Title of={IconStories} />
+<Description of={IconStories} />
+
+## Icon sets
+
+The SVG icons used in Spectrum CSS are a part of two different icon sets, "workflow" and "ui". The two sets have different uses and methods of sizing.
+
+### Workflow icons
+
+The workflow icon set contains a several hundred icons to choose from.
+These icons can be seen in use within Button, Action button, Menu, and many other components. 
+Here is an example with just a few of these icons:
+
+<Canvas of={IconStories.WorkflowDefault} />
+
+These icons use "t-shirt" sizes (e.g. small, medium), that are the same width and height for each icon in the set:
+
+<Canvas of={IconStories.WorkflowSizing} />
+
+### UI icons
+
+UI icons are atomic pieces (e.g., arrows, crosses, etc.) that are used as part of some components. The chevron within
+the Combobox component is one example.
+
+Unlike workflow icons, each UI icon comes in specific numbered sizes. They do not use "t-shirt" sizing (i.e. medium, small).
+They use their specific token defined sizes in various number size classes. They have unique classes applied that set their size in CSS. For example:
```

Comment:
I find this a bit confusing:
> They use their specific token defined sizes in various number size classes

Do you think it's needed? I wonder if the following sentence and examples cover what this is trying to say:

```suggestion
They have unique classes applied that set their size in CSS. For example:
```